### ISSUE TITLE
🐛 Secrets store: secrets related env var are not declared in Kubernetes manifests

### ISSUE THREAD

**Issue by alafanechere** [airbytehq/airbyte#9847]
The following environment vars are not defined in Kubernetes manifests we have (both Kustomize and Helm):

* `SECRET_STORE_GCP_PROJECT_ID`
* `SECRET_STORE_GCP_CREDENTIALS`
* `SECRET_PERSISTENCE`

This prevents our Kubernetes users to use a GSM secret store.

[Slack discussion here](https://airbytehq-team.slack.com/archives/C019WJFR1JT/p1642764265216300)

**Comment by supertopher** [comment]
This is probs done.  Please check if it is and close this task if it is already done

**Comment by supertopher** [comment]
before starting work on this please confirm if this is done or not.

@xpuska513 I think you may have context on this

**Comment by tkorenko** [comment]
_It would be good to have more context in task description..._
